controller/main.go

```diff
@@ -28,7 +28,7 @@ import (
 	"go.universe.tf/metallb/internal/version"
 
 	"github.com/go-kit/kit/log"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Your editor is doing this automatically most likely. Personally I don't mind merging this change but I wanted to verify this is intentional.